pilot/AccountTableViewController.swift

```diff
@@ -27,22 +27,19 @@ class AccountTableViewController: UITableViewController {
   // MARK: - TableViewDataSource
 
   override func numberOfSections(in tableView: UITableView) -> Int {
-    return 2
+    return 1
   }
 
   override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
     switch section {
-    case 0: return 2
-    case 1: return 1
+    case 0: return 1
     default: fatalError("Unknown number of sections")
     }
   }
 
   override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
     switch indexPath.section {
```

Comment:
Here it looks kind of weird that there is only case 0 and default, especially because there are two switch-cases with only two options. Seems better suited for a `guard indexPath.section == 0 else fatalError(...)` ?